asciinema/tests/test_helper.py

```diff
@@ -0,0 +1,20 @@
+import sys
```

Comment:
This file was also moved and changed, so the diff is not representative, not sure why github is not showing in a similar manner as test_pty. It would be better to compare the current `tests/test_helper.py` with it for a proper diff.